kafka/kafkatest/utils.go

```diff
@@ -63,11 +64,12 @@ func (f *FakeKafkaProducer) Close() error {
 
 func NewFakeKafkaConsumer(log logrus.FieldLogger, distri <-chan *kafkalib.Message) *FakeKafkaConsumer {
 	r := &FakeKafkaConsumer{
-		messages: make([]*kafkalib.Message, 0),
-		offset:   0,
-		notify:   make(chan struct{}),
-		log:      log,
-		commits:  make(chan *kafkalib.Message, 1000),
+		messages:         make([]*kafkalib.Message, 0),
+		offset:           0,
+		uncommitedOffset: 0,
```

Comment:
"uncommitted" seems wrong if this is tracking the read unconnected to the committed offsets now?, `readOffset` ?

Reply:
Yah, I'm not super good with Kafka terminology. If read makes more sense, I'm 👍 .